### DISCUSSION TITLE
Accuracy in MW frequencies

### DISCUSSION THREAD

**Discussion by imoradi** [eradiate/eradiate#380] [Q&A]
Does this model work for microwave frequencies? If so does it include cloud scattering information for the MW region? Lastly, would be possible to output some intermediate parameters such as atmospheric absorption coefficient and cloud scattering per each layer? Thank you

**Comment by leroyvn** [comment]
Hi @imoradi, I'll address your questions point by point.

> Does this model work for microwave frequencies? 

Eradiate is currently focusing on the solar reflective region, meaning that the microwave region is unfortunately out of its scope.

> Would be possible to output some intermediate parameters such as atmospheric absorption coefficient and cloud scattering per each layer? 

Atmospheric collision coefficients and single-scattering albedo can be retrieved using the [`Atmosphere.eval_radprops()`](https://eradiate.readthedocs.io/en/stable/rst/reference_api/generated/autosummary/eradiate.scenes.atmosphere.HeterogeneousAtmosphere.html#eradiate.scenes.atmosphere.HeterogeneousAtmosphere.eval_radprops) function. An important caveat is that in CKD mode, these coefficients are output as a function of the spectral coordinate *g*, which then has to be processed to apply the spectral quadrature rule. This process is unfortunately not automated yet.

The scattering phase function can be retrieved from the [`Atmosphere.phase`](https://eradiate.readthedocs.io/en/stable/rst/reference_api/generated/autosummary/eradiate.scenes.atmosphere.Atmosphere.html#eradiate.scenes.atmosphere.Atmosphere.phase) property. In the most general case, this will return a `BlendPhaseFunction` instance, which then has to be evaluated with additional programming I cannot detail quickly here.

The bottom line is that you can do all this, but we still have to automate all of this a bit more to make it easier for you.
